xcode/Subconscious/Shared/Components/Common/StoryComboView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  StoryComboView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 24/9/2022.
+//
+
+import SwiftUI
+
+/// A story is a single update within the FeedView
+struct StoryComboView: View {
+    var story: StoryCombo
+    var action: (EntryLink, String) -> Void
+    
+    func synthesize() {
+        let slug = "combo/\(story.entryA.slug.description)/\(story.entryB.slug.description)"
```

Comment:
We should probably come up with a convention for Geist-generated content like this. A couple ideas on possible conventions, without anchoring on any specific one:

- I like what you're doing here with `/combo/slug/slug`. OTOH, by default I lean toward flat naming schemes, since they create more chance of name collision (good, since it causes you to return to old notes and add to them https://subconscious.substack.com/p/knowledge-gardening-is-recursive).
- I like the "A-B" conjoining you're doing here. It makes sense that if we already created a combo, we might want to see that content instead of generating new content? How do we feel about `slug-slug` rather than `combo/slug/slug`?
- For Geist-generated content where we want a low chance of collision, maybe we default to `YYYY-MM-DD-name`? Not sure we need that here, though.

Reply:
I feel similarly torn. Nesting under `/combo` avoids polluting the root namespace and helps trace the origin of the entry, but the argument of maximising name collision is a strong one. 

If tracing the origin is actually useful we could also include `source: [[ComboGeist]]` in the content instead of  prefixing with `/combo`? I can see good arguments for `A-B` naming but longer slugs might be difficult to visually parse this way.

Also occurs to me that the same two entries (A and B) could be combined in A-B order or B-A order and I feel intuitively that those should point to the same combined slug? Perhaps instead of blindly concatenating A and B we should sort them them alphanumerically by slug beforehand (or maybe creation date?) I'll push a commit to try this out.

Having a defined ordering on slugs exposes some additional meaning from the `/combo/entry-a/entry-b` format, since viewing `/combo/entry-a` would list all the entries where you've combined `entry-a` with something else. 

e.g.
<img width="372" alt="image" src="https://user-images.githubusercontent.com/5009316/193160879-961e2d62-f8d8-4e1c-8bbf-792eff2eed3f.png">

What's our orienting principle here? If we're maximising naming collisions then I vote to drop `combo/` and flatten to `slug-slug` but if we're maximising explorability I would vote to keep it.

Reply:
@bfollington let's go with the following:

- Order slugs by alpha
- combine as `slug-slug` (using `Slug(formatting: "\(a) \(b)")` should do the trick)